pex/variables.py

```diff
@@ -327,6 +327,17 @@ def PEX_IGNORE_RCFILES(self):
     """
     return self._get_bool('PEX_IGNORE_RCFILES', default=False)
 
+  @property
+  def PEX_EMIT_WARNINGS(self):
```

Comment:
This will update https://pex.readthedocs.io/en/stable/api/index.html#pex-variables-module on the next release, right?

Reply:
No. See #428. I'm hoping @wickman will hand over the keys. I'm sure he'd gladly do so but he's just hard to get a hold of.